packages/client/src/project/entities.ts

```diff
@@ -13,6 +13,15 @@ export function getProjectEntities (projectId: string, apiClient: AxiosInstance)
       const entities = res.data;
       return wrapEntityArray(entities);
     },
+    async put(entity: Entity): Promise<WrappedEntity> {
+      const entityId = entity._id;
+      delete entity._id;
```

Comment:
Don't mutate an argument provided by the user. This might lead to bugs because the user will not be aware that their object was mutated and a property mysteriously disappeared.

Consider doing something like
```js
const _entity = { ...entity } // create copy of argument
delete _entity._id; // mutate local copy without affecting the original argument
```
note that `{...entity}` creates a shallow copy and does not protect against accidentally mutating nested properties. But in this case it's not a problem since you're only modifying the root-level `_id` property.

I also suggest that you create a test for this function, and the test should ensure that the input is not mutated. One way to test for that is to wrap the input with `Object.freeze` before passing it to the function, and if it's mutated, an error will be thrown and the test will fail.